docs/annotation-cheat-sheet.md

```diff
@@ -0,0 +1,161 @@
+# Annotation Cheat-Sheet
+
+How to construct an annotation from a term definition, or re-engineer a term definition from an annotation example?
+
+Here's how.
+
+## Vocabularies
+
+Terms are defined within a "vocabulary", which is just an EDMX document.
+
+```xml
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:DataServices>
+
+    <Schema Namespace="Vocab"
+
+      xmlns="http://docs.oasis-open.org/odata/ns/edm">
+```
+
+To use a term in an annotation, the vocabulary of the term needs to be referenced and included.
+
+```xml
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="https://somewhere.org/vocab.xml">
+
+    <edmx:Include Namespace="Vocab" />
+
+  </edmx:Reference>
+```
+
+## Primitive Terms
+
+If the term has a primitive type
+
+```xml
+<Term Name="StringTerm" Type="Edm.String" />
+```
+
+A constant annotation value is provided in a [corresponding attribute](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_ConstantExpression).
```

Comment:
actually: "may be provided"; alternatively, element notation can be used as well